src/main/ml-modules/root/config/searchResultsSortConfig.mjs

```diff
@@ -1,19 +1,55 @@
+import op from '/MarkLogic/optic';
+const crm = op.prefixer('http://www.cidoc-crm.org/cidoc-crm/');
+const la = op.prefixer('https://linked.art/ns/terms/');
+const lux = op.prefixer('https://lux.collections.yale.edu/ns/');
+const skos = op.prefixer('http://www.w3.org/2004/02/skos/core#');
+
 const SORT_BINDINGS = {
   agentActiveDate: {
     indexType: 'field',
     indexReference: 'agentActiveStartDateLong',
     defaultOrder: 'asc',
   },
+  agentClassificationConceptName: {

```

Comment:
Just noting a couple configuration differences.  In searchTermsConfig.mjs, we specify the predicates as strings and support multiple per term; e.g., `predicates: ['lux("setClassifiedAs")']`.  Strings make sense for search terms as we generate CTS string before evaluating.  Herein, we get a single resolved one.  Not sure if it would make sense to support multiple per binding.  Regardless, I'm not aware of any such requirement today.

1